setup.py

```diff
@@ -86,6 +86,7 @@ def read(fname):
     keywords='tests py.test pytest fixture redis mongo mongodb'
              ' rabbit rabbitmq mysql postgres postgresql'
              ' elasticsearch',
+    license="GNU Lesser GPL",
```

Comment:
Something like `GNU LGPLv3+`? Version number and "or later" are important. I don't know how this field usually looks like.